components/script_traits/lib.rs

```diff
@@ -40,8 +40,8 @@ use gfx_traits::LayerId;
 use ipc_channel::ipc::{IpcReceiver, IpcSender};
 use libc::c_void;
 use msg::constellation_msg::{ConstellationChan, Failure, PipelineId, WindowSizeData};
-use msg::constellation_msg::{Key, KeyModifiers, KeyState, LoadData, MouseButton};
-use msg::constellation_msg::{MozBrowserEvent, PipelineNamespaceId, SubpageId};
+use msg::constellation_msg::{Key, KeyModifiers, KeyState, LoadData, SubpageId};
```

Comment:
This `SubpageId` import should come on the next line. Our style checker likes imports to be in alphabetic order